ts-obs/cmd/kubectl.go

```diff
@@ -0,0 +1,339 @@
+package cmd
+
```

Comment:
For a whole bunch of these functions I wonder if the code would be cleaner if instead fo getting the namespace and name as parameters, it would be better to do the following inside these functions.

Would that remove a lot of code duplication?

 ```	var name string
	name, err = cmd.Flags().GetString("name")
	if err != nil {
		return fmt.Errorf("could not find release name: %w", err)
	}

	var namespace string
	namespace, err = cmd.Flags().GetString("namespace")
	if err != nil {
		return fmt.Errorf("could not get namespace: %w", err)
	}```